src/functions-and-arrays.js

```diff
@@ -1,41 +1,53 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+const maxOfTwoNumbers = (a, b) => a > b ? a : b
 
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+const findLongestWord = (arr) => {
+  let longestWord = arr[0];
+  arr.length === 0 ? longestWord = null : longestWord = arr[0];
+  arr.forEach(x => x.length > longestWord.length ? longestWord = x : x)
+  return longestWord;
+}
```

Comment:
- Los operadores ternarios (`?`) son muy útiles en operaciones de cortocircuito en el que hay varias opciones, pero como en tu caso la segunda opción es la misma que la situación inicial, con un `if` te basta y te sobra:

```suggestion
const findLongestWord = (arr) => {
  let longestWord = arr[0];
  if (arr.length === 0) longestWord = null;
  arr.forEach(x => x.length > longestWord.length ? longestWord = x : x)
  return longestWord;
}
```